azure-stack/hci/concepts/storage-spaces-direct-overview.md

```diff
@@ -24,7 +24,7 @@ Storage Spaces Direct is a software-defined storage solution that allows you to
 
 You can scale out the storage capacity of your cluster by adding more drives or adding more servers in the cluster. Storage Spaces Direct automatically onboards the new drives and rebalances the storage pool. It also automatically uses the fastest storage media present to provide built-in and always-on cache.
 
-Storage Spaces Direct is a core technology of Azure Stack HCI, versions 21H2 and 20H2. It’s also included in the Datacenter editions of Windows Server 2022, Windows Server 2019, Windows Server 2016, [Windows Server Insider Preview Builds](https://insider.windows.com/for-business-getting-started-server/) and Azure editions of Windows Server 2022 and Windows Server 2019.
+Storage Spaces Direct is a core technology of Azure Stack HCI, versions 21H2 and 20H2. It’s also included in the Datacenter editions of Windows Server 2022, Windows Server 2019, Windows Server 2016, [Windows Server Insider Preview Builds](https://insider.windows.com/for-business-getting-started-server/) and Azure editions of Windows Server 2022.
```

Comment:
@tksh164: Thanks for your contribution. Can you also replace "editions" with "Edition"? Thanks!

Reply:
@ManikaDhiman Done!